libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -148,6 +148,24 @@ const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_IGN_WID6", 15, AP_Proximity, _ignore_width_deg[5], 0),
 
+    // @Param: _DISTMAX_M
+    // @DisplayName: maximum distance (in meters) of sensor
+    // @Description: maximum distance (in meters) of sensor
+    // @Units: m
+    // @Range: 0.00 25.00
+    // @Increment: 0.01
+    // @User: Standard
+    AP_GROUPINFO("_DISTMAX_M", 19, AP_Proximity, _distance_max[0], 16.00f),
+
+    // @Param: __DISTMIN_M
+    // @DisplayName: minimum distance (in meters) of sensor
+    // @Description: minimum distance (in meters) of sensor
+    // @Units: m
+    // @Range: 0.00 25.00
+    // @Increment: 0.01
+    // @User: Standard
+    AP_GROUPINFO("__DISTMIN_M", 20, AP_Proximity, _distance_min[0], 0.20f),
```

Comment:
double underscore I think and normally we don't include the units in the name..

Reply:
@rmackay9 san.. Comments thanks. I'll fix it.